src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/_help.py

```diff
@@ -140,6 +140,11 @@
             short-summary: Manage protection of your items, enable protection or disable it, or take on-demand backups.
             """
 
+helps['backup protection isenabled-for-vm'] = """
```

Comment:
I would recommend `exists` or `check`.  In the future you would presumably be able to check if a SQL server is backed up, etc.  There's no need for a bunch of "isenabled-for-foo" commands.

Reply:
Per our meeting, please use the name "check-vm"